trunk/SUAVE/Components/Energy/Converters/Expansion_Nozzle.py

```diff
@@ -155,6 +157,9 @@ def compute(self,conditions):
         Mach[i_high]  = 1.0*Mach[i_high]/Mach[i_high]
         P_out[i_high] = Pt_out[i_high]/(1.+(gamma[i_high]-1.)/2.*Mach[i_high]*Mach[i_high])**(gamma[i_high]/(gamma[i_high]-1.))
         
+        # A cap to make sure Mach doesn't go to zero:
+        Mach[Mach<=0.0] = 0.001
```

Comment:
I think it would be more appropriate to raise an error if this happens. Having assignments like this buried in the code can be confusing when trying to debug.

Reply:
I added a warning, there are similar warnings in other turbomachinery components.